OMF_APITests/UnitTest1.cs

```diff
@@ -1,14 +1,249 @@
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using OMF_API;
 using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
 using Xunit;
 
 namespace OMF_APITests
 {
     public class UnitTest1
     {
+        private static readonly HttpClient client = new HttpClient();
+
         [Fact]
         public void Test1()
         {
-            Assert.True(OMF_API.Program.runMain(true));
+            // Steps 1 to 7 - Run the main program
+            Dictionary<string, dynamic> sentData = new Dictionary<string, dynamic>();
+            Assert.True(OMF_API.Program.runMain(true, sentData));
+            // Step 8 - Check Creations
+            Assert.True(checkCreations(sentData));
+            // Step 9 - Cleanup
+            Assert.True(cleanup());
+        }
+
+        private bool checkCreations(Dictionary<string, dynamic> sentData)
+        {
+            AppSettings settings = OMF_API.Program.getAppSettings();
+            IList<Endpoint> endpoints = settings.Endpoints;
+            dynamic omfTypes = OMF_API.Program.getJsonFile("OMF-Types.json");
+            dynamic omfContainers = OMF_API.Program.getJsonFile("OMF-Containers.json");
+            dynamic omfData = OMF_API.Program.getJsonFile("OMF-Data.json");
+
+            bool success = true;
+
+            foreach (var endpoint in endpoints)
+            {
+                try
+                {
+                    if (string.Equals(endpoint.EndpointType, "PI"))
```

Comment:
Use the `string.Equals()` overload that specifies a case insensitive `StringComparisonType`